tests/queries/tests.py

```diff
@@ -1629,6 +1630,21 @@ def test_ordering(self):
             [2, 1, 3]
         )
 
+        # newlines control for raw sql parse of compiler.get_order_by
```

Comment:
Please use a more appropriate location for the test. `tests/expressions/tests.py` seems like a better candidate.

Reply:
I've moved the test into `tests/expressions/tests.py`, thanks.